changelog/12960.breaking.rst

```diff
@@ -0,0 +1 @@
+Test functions containing a yield now cause an explicit error. They have not been run since Pytest 4.0, and were previously marked as an expected failure and deprecation warning.
```

Comment:
There's a section named `yield tests` in `deprecations.rst`, under `Deprecated Features`. Given this is actually being removed, please move that section to `Removed Features and Breaking Changes`, adding a `.. versionremoved:: 8.4` to it (see the others for an example).